cluster/gce/gci/configure-helper.sh

```diff
@@ -2065,6 +2065,12 @@ function start-kube-controller-manager {
   if [[ -n "${RUN_CONTROLLERS:-}" ]]; then
     params+=("--controllers=${RUN_CONTROLLERS}")
   fi
+  if [[ -n "${KUBE_CONTROLLER_MANAGER_API_QPS:-}" ]]; then
```

Comment:
kube-up is generally deprecated.

You cal already pass this information in a bit more hack way, as we do in our tests:
https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-scalability/sig-scalability-presets.yaml#L270

I'm wondering how important is it to add new explicit params for it...

Reply:
Ah, interesting. In that case I think this PR is redundant. I don't think we need this kube-up change since kubetest has a way to set the ctrlmgr/scheduler test args. Closing this PR and updating the other PR.